spring.factories problem with low version spring-boot

### Integration

sentry-spring-boot-starter

### Java Version

8

### Version

6.1.4

### Steps to Reproduce

There is an extra space character in sentry-spring-boot-starter/spring.factories
I have a historical project use spring-boot 2.0.1. There is an exception thrown when startup.

```
Caused by: java.io.FileNotFoundException: class path resource [ io/sentry/spring/boot/SentryWebfluxAutoConfiguration.class] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:180)
	at org.springframework.core.type.classreading.SimpleMetadataReader.<init>(SimpleMetadataReader.java:51)
	at org.springframework.core.type.classreading.SimpleMetadataReaderFactory.getMetadataReader(SimpleMetadataReaderFactory.java:103)
```

It seems low version spring-boot don't trim line in spring.factories. It works in high version spring-boot.

### Expected Result

remove extra space character

### Actual Result

remove extra space character

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Fixes
+
+- Remove extra space from `spring.factories` causing issues in old versions of Spring Boot ([#2181](https://github.com/getsentry/sentry-java/pull/2181))
+
 ## 6.3.0
 
 ### Features

--- a/sentry-spring-boot-starter/src/main/resources/META-INF/spring.factories
+++ b/sentry-spring-boot-starter/src/main/resources/META-INF/spring.factories
@@ -1,4 +1,4 @@
 org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
 io.sentry.spring.boot.SentryAutoConfiguration,\
-io.sentry.spring.boot.SentryLogbackAppenderAutoConfiguration, \
+io.sentry.spring.boot.SentryLogbackAppenderAutoConfiguration,\
 io.sentry.spring.boot.SentryWebfluxAutoConfiguration

